src/AcademicProfileCommentSchema.js

```diff
@@ -20,6 +20,7 @@ module.exports = (conn) => {
       enum: [
         'tech',
         'soft',
+        'general',
```

Comment:
Good, even though this is considered an `enum value` rather tan a property (which is `type` in this case). Please describe this in your commit message as well.